package.json

```diff
@@ -100,7 +103,7 @@
     "protractor": "~5.1.0",
     "rimraf": "^2.6.1",
     "rxjs": "^5.1.0",
-    "ts-node": "~2.0.0",
+    "ts-node": "~3.3.0",
```

Comment:
I see no reason to bump up the `ts-node` version, everything will still work with `~2.0.0`.

Reply:
See my other comment. This would ideally be part of another Pull Request that fixes a new issue (something like "cannot compile library in Windows") :wink:

Reply:
I agree. Reverted :)